class UserAlreadyVerifiedException(Exception):
    pass


class UserDoesNotExistException(Exception):
    pass
